### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Disk_PasteDisk_ScreenReader: Jaws does not read the title of the dialog as Paste disk when opened.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#2093]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;#MAS2.4.2 #JAWS; #Win32; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
JAWS 2019
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to Paste Disk button and click on it.
7. "Paste Disk" Popup gets opened.
8. Open JAWS and check whether it reads the title of the dialog when opened.
​Actual:​
Jaws does not read the title of the dialog as Paste disk when opened.
**Same issue is observed throughout the application wherever dialog present.**
**Expected:**
Jaws should read the title of the dialog as Paste Disk.
**UserImpact:**
Screen reader users unable to get that which dialog opened.

**Comment by MRayermannMSFT** [comment]
@nirmalshaoo , I'm having trouble repro'ing this issue. When I use JAWS, it reads the title of the dialog. Please see this attached screenshot of my JAWS log:
![image](https://user-images.githubusercontent.com/21112697/69367045-574e0480-0c4c-11ea-8c4f-f6fa754d3f50.png)

**Comment by MRayermannMSFT** [comment]
Even if I don't open JAWS until after opening the paste dialog, JAWS still announces the title of the dialog. 
![image](https://user-images.githubusercontent.com/21112697/69368112-54ecaa00-0c4e-11ea-90e7-7af59db0a72a.png)

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo can you please follow up?

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo can you please follow up?

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo can you please follow up?

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Apologies for late response.
Verified the issue in JAWS version: "2019.1909.28 ILM"  still able to repro the issue.
When the dialog opens, Jaws reads like below:
**Actual**: "Microsoft Azure Storage Explorer Cancel button to activate press Space bar This page has no links Microsoft Storage Explorer-Paste Disk Paste Disk and reads the entire page content present in screen".
So here screen reader users may get confused about understanding the title of the dialog and not be able to know the exact title of the dialog.

**Expected**: When dialog gets opened Jaws should announce exact title of the dialog like 
"Microsoft Azure Storage Explorer - Paste Disk Window"  and it should not announce the entire content present in the screen in one attempt as soon as the screen gets opened.

**Comment by nirmalsahoo** [comment]
Verified this issue in version:1.11.2, JAWS still announces like below:
"Microsoft Azure Storage Explorer Cancel button to activate press Space bar This page has no links Microsoft Storage Explorer-Paste Disk Paste Disk and reads the entire page content present in screen".

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo which part exactly is the problem? Here's my JAWS logs for opening the dialog and doing nothing until it is done speaking:

```
Paste Button 
Paste the copied disks
```
I hit the enter button here
```
Microsoft Azure Storage Explorer 
Page has no links
Microsoft Azure Storage Explorer - Paste Disk 
Microsoft Azure Storage Explorer - Paste Disk
Microsoft Azure Storage Explorer - Paste Disk
Paste Disk
Choose the name for the disk and and then specify the remaining parameters.
Disk name:
Edit 4gbtest3-reupload
Region:
Region: Combo box 
East US
Availability zone:
Availability zone: Combo box 
None
Account type:
Account type: Combo box 
Premium SSD
Paste
Button 
Cancel Button 
Cancel
```

- From what I can tell, JAWS reading the entire page is a feature of JAWS, it is not something we can control.
- I can hear and see the title of the dialog being announced.

So what is wrong?

**Comment by MRayermannMSFT** [comment]
I found this in the JAWS documentation:
![image](https://user-images.githubusercontent.com/21112697/71929272-181d3980-314e-11ea-96db-7a71282fa694.png)
Please note that Storage Explorer is an Electron application. This means that screen readers will commonly view our UI as a webpage. We've discussed this before with Ryan, and he agrees that we shouldn't do anything to avoid that happening.

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo, I'm going to close this and mark as needs verification. Please answer my questions. Thank you.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified the issue and is still reproduced on version 1.12.0.Please refer the attachment 2093.pptx
[2093.pptx](https://github.com/microsoft/AzureStorageExplorer/files/4148470/2093.pptx)

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo I cannot hear the screen reader in your video. See my attached file for what I am hearing.


[2020-02-03 10-12-27.zip](https://github.com/microsoft/AzureStorageExplorer/files/4149380/2020-02-03.10-12-27.zip)

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Apologies for the attachment. Referring to 2020-02-03 10-12-27.zip attachment and verified on version 1.12.0 and JAWS "2019.1912.11 ILM",Issue is no longer reproduced,hence close this issue.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
